recipe/bld.bat

```diff
@@ -0,0 +1,38 @@
+@ECHO ON
+
+@REM I cannot for the life of me figure out how Cygwin/MSYS2 figures out its
+@REM root directory, which it uses to find the /etc/fstab which *sometimes*
+@REM affects the choice of the cygdrive prefix. But, regardless of *why*,
+@REM I find that we need this to work:
+mkdir %BUILD_PREFIX%\Library\etc
+echo none / cygdrive binary,user 0 0 >%BUILD_PREFIX%\Library\etc\fstab
+echo none /tmp usertemp binary,posix=0 0 0 >>%BUILD_PREFIX%\Library\etc\fstab
```

Comment:
Instead of any of this, add a dependency of `m2-filesystem`.

Reply:
Will do.